### PR TITLE
Migrate generator package

### PR DESCRIPTION
**Pull request by beerose**
Closes: https://github.com/blitz-js/blitz/issues/3217

## Bug Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)

## Feature Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)
- [ ] Documentation added/updated (submit PR to [blitzjs.com repo `canary` branch](https://github.com/blitz-js/blitzjs.com/tree/canary))

### CODE REVIEW

File: packages/blitz/package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "express": "4.17.3",
     "http": "0.0.1-security",
     "node-fetch": "3.2.3",
-    "npm-which": "3.0.1",
+    "npm-which": "^3.0.1",
```
Comment (noxify) [inline packages/blitz/package.json:35]: Maybe I'm a bit paranoid after `faker.js` and `node-ipc`, but wouldn't it be better to use the absolute version ( w/o `^` )?

**Review by money5240 — APPROVED** [review]
packages/blitz-auth/src/server/auth-plugin.ts

**Review by money5240 — APPROVED** [review]
packages/blitz/package.json

File: packages/blitz/package.json (hunk 1 above)
Comment (beerose) [reply packages/blitz/package.json]: Yeah, good point

**Pull request merged**